functions/stats.ts

```
@@ -0,0 +1,298 @@
+  let streakEndDate: Date | null = null;
+  let onActualStreak: boolean = false;
+  const oneDay: number = (24 * 60 * 60 * 1000) * 1;
+  for (const td of tagDates) {
+    if (!onActualStreak) {
+      streakStartDate = td;
+      streakEndDate = td;
+    }
+    const index: number = tagDates.indexOf(td);
+    if (index === 0) {
+      // Need a previous date to compare with, skip the first
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)